### PR TITLE
[release-v1.11] Backport fix for random consumer group id access

### PR DESCRIPTION
**Pull request by pierDipi**
This is the non breaking change related to https://github.com/knative-extensions/eventing-kafka-broker/commit/bfeb3c39f741b8380788cd573bff499c59834e47

### CODE REVIEW

**Review by creydr — APPROVED** [review]
/lgtm

File: control-plane/pkg/reconciler/trigger/trigger.go [hunk 1]
```diff
@@ -488,3 +489,8 @@ func deliveryOrderFromString(val string) (contract.DeliveryOrder, error) {
 		return contract.DeliveryOrder_UNORDERED, fmt.Errorf("invalid annotation %s value: %s. Allowed values [ %q | %q ]", deliveryOrderAnnotation, val, sources.Ordered, sources.Unordered)
 	}
 }
+
+func isGroupAuthorizationError(err error) bool {
```
Comment (matzew) [inline control-plane/pkg/reconciler/trigger/trigger.go:493]: I like to filter "auth" issues out, for a smooth transition towards the actual upstream logic in 1.13

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**